server/hardware.c

```diff
@@ -324,6 +324,13 @@ void hw_get_start_ts(struct timespec *ts)
     ts->tv_nsec = (typeof(ts->tv_nsec)) compat_ts.tv_nsec;
 }
 
+
+void hw_get_hw_start_ts(struct timespec *ts)
+{
+    ts->tv_sec = (time_t) read_named_register(PCAP_TS_SEC);
+    ts->tv_nsec = (typeof(ts->tv_nsec)) read_named_register(PCAP_TS_NSEC);
```

Comment:
And this will have to be multiplied by `1 / read_named_register(NOMINAL_CLOCK)`

Reply:
Done